### PR TITLE
Request invite

### PR DESCRIPTION
**Pull request by zevav**

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -2,3 +2,5 @@
 
 SPEECHMATICS_API_KEY = _local_config.SPEECHMATICS_API_KEY
 SECRET_KEY = _local_config.SECRET_KEY
+SLACK_TOKEN = _local_config.slack_token
```
Comment (solumos) [inline app/config.py]: These new credentials need to be added to _local_config

File: app/routes.py [hunk 2]
```diff
@@ -27,3 +30,68 @@ def homepage():
...
+                                'person_id':person_id,
+                                'stage_id':'6',
+                                'org_id':org_id})
+        if not json.loads(add_deal.text)['success']:
+            print "there was a problem adding the " + prospect_name + " deal to Pipedrive."
+            print add_deal.reason
+        else:
+            print "The deal ({}) was added to Pipedrive.".format(prospect_name)
+        flash("Thanks for your interest!  We'll be in touch shortly.")
+    return redirect(url_for('homepage'))
```
Comment (solumos) [inline app/routes.py]: This is a lot of controller logic for a route

File: app/templates/index.html.orig [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+           $("html, body").animate({ 
+           scrollTop: $("#youtube").offset().top 
+           }, 5);
+        });
+    </script>
+    {% endif %}
+
+</body>
+
+</html>
```
Comment (solumos) [inline app/templates/index.html.orig]: What is this file?

File: app/templates/index.html [hunk 4]
```diff
@@ -3,196 +3,242 @@
...
-                </div>
-            </div>
+      </div><!--modal-dialog-->
+    </div><!--request_invite-->
+  </div>
+  </aside>
+  </header>
+
+  {% block youtube %}
+  {% endblock %}
```
Comment (solumos) [inline app/templates/index.html]: Not sure that this block still belongs here

Comment (zevav) [inline app/templates/index.html]: doesn't belong -- it's probably commented out?

Zev Averbach
Averbach Transcription
(212) 729-4850
www.avtranscription.com

**Pull request merged**